lib/Reader.php

```diff
@@ -22,6 +22,12 @@ class Reader extends XMLReader {
     use ContextStackTrait;
 
     /**
+     * Reading indicator
+     * @var bool
+     */
+    public $reading = false;
```

Comment:
protected would be suitable when you can describe a usefull case in which you need it ;-).

if you can't -> private.